events/events.go

```diff
@@ -0,0 +1,308 @@
+package events
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"github.com/SAP/service-fabrik-cli-plugin/errors"
+	"github.com/SAP/service-fabrik-cli-plugin/helper"
+	"github.com/cloudfoundry/cli/plugin"
+	"github.com/fatih/color"
+	"github.com/olekukonko/tablewriter"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+var (
+	httpClientDisabledSecurityCheck *http.Client
+	httpClientEnabledSecurityCheck  *http.Client
+)
+
+type EventCommand struct {
+	cliConnection plugin.CliConnection
+}
+
+func initialize() {
+	httpClientDisabledSecurityCheck = createHttpClient(true)
+	httpClientEnabledSecurityCheck = createHttpClient(false)
+}
+
+func NewEventsCommand(cliConnection plugin.CliConnection) *EventCommand {
+	command := new(EventCommand)
+	command.cliConnection = cliConnection
+	return command
+}
+
+const (
+	red                color.Attribute = color.FgRed
+	green              color.Attribute = color.FgGreen
+	cyan               color.Attribute = color.FgCyan
+	white              color.Attribute = color.FgWhite
+	MaxIdleConnections int             = 25
+	RequestTimeout     int             = 180
+)
+
+func AddColor(text string, textColor color.Attribute) string {
+	printer := color.New(textColor).Add(color.Bold).SprintFunc()
+	return printer(text)
+}
+
+type Configuration struct {
+	ServiceBroker       string
+	ServiceBrokerExtUrl string
+	SkipSslFlag         bool
+}
+
+func findNextPage(output []string) string {
```

Comment:
Move this to a common file.

Just a suggestion here, I am fine either way (with helper file or with utils file):
We can create another file called `utils` and place all these common code there. Since, the helper package only contains `config-json-parser` and this (and many other functions suggested, like http functions) probably don't fall under the same either.